### DISCUSSION TITLE
Job runs indefinitely

### DISCUSSION THREAD

**Discussion by macgills** [community/community#25522] [Actions]
<p>I am implementing a github action instrumentation proof of concept for an android project and sometimes my jobs don’t complete, see <a href="https://github.com/kiwix/kiwix-android/runs/347375883" rel="nofollow noopener">here.</a></p>
<p>CoverageReport is the job and the afflicted versions in the build matrix are 27/28/29 but I think this is just an artifact of the ordering. I can not view the logs for the - I assume - failing step and have no way to diagnose this. A usually 10min step is at 30+ so something must have gone quite wrong.</p>

**Comment by Yanjingzhu** [comment]
<p>Do you still meet this issue now? Based on the error message on annotations of coverageReport(27), <em>The runner: GitHub Actions 9 lost communication with the server. Verify the machine is running and has a healthy network connection.</em></p>
<p>Was your network not stable during the execution of this workflow run?  If GitHub Actions lost communication with the server, the server could not post back logs to the console.  </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/9/9744403832c7e51c81df2ca13bb029edb858c3ea.png" data-download-href="/uploads/short-url/lAak8D2Xez1y1aPBEtAz0haiT6G.png?dl=1" title="annotation.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181091715-c6860ec4-925c-4c86-a8ca-35f0958d5d30.png" alt="annotation.png" data-base62-sha1="lAak8D2Xez1y1aPBEtAz0haiT6G" width="690" height="249" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/9/9744403832c7e51c81df2ca13bb029edb858c3ea_2_690x249.png, https://aws1.discourse-cdn.com/github/optimized/2X/9/9744403832c7e51c81df2ca13bb029edb858c3ea_2_1035x373.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/9/9744403832c7e51c81df2ca13bb029edb858c3ea_2_1380x498.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/9/9744403832c7e51c81df2ca13bb029edb858c3ea_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">annotation.png</span><span class="informations">1896×686 35.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by macgills** [comment, marked as answer]
<p>I think it might have been an update to a github action, I was using v2, specifying v2.0.0 worked</p>
